docs/manual/manual_safety_systems.rst

```diff
@@ -266,6 +266,12 @@ The following examples are meant to illustrate the effect that slew limiting has
 As you can see, the slew limit parameter helps protect against sudden changes in commanded voltage. This can be especially useful in :ref:`noisy/vibration heavy 
 environments <vibration_and_jittering>`. The amount of slew filtering that is necessary (if any) is completely application specific, and will require manual tuning.
 
```

Comment:
I might move this up from the very bottom of this section so its a bit more visible. People may not actually make down this far.

Maybe put it right before the examples bit, e.g. before "The following examples..." to increase the chance of people seeing it